components/calendar/doc/index.zh-CN.md

```diff
@@ -34,7 +34,7 @@ registerLocaleData(zh);
   <div *dateCell>Foo</div>
 </nz-calendar>
 <!-- 传入 TemplateRef 的方式 -->
-<ng-template #dateCellTpl>Bar</ng-template>
+<ng-template #dateCellTpl let-date><span>{{ date.getDate() }}</span></ng-template>
```

Comment:
`index.en-US.md` also needs to be this change

Reply:
How about using `{{ date | date:'d'}}`? `DatePipe` is `pure: true` and avoids the re-invocation. Also calling function in template is not recommended.

Reply:
sure,it's better.